core/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -161,6 +162,16 @@
   public abstract T usePlaintext(boolean skipNegotiation);
 
   /**
+   * Makes the client use TLS.
+   *
+   * @return this
+   * @throws UnsupportedOperationException if transport security is not supported.
+   */
+  public T useTransportSecurity() {
```

Comment:
We mark new APIs with `@ExperimentalApi("https://link/to/github/issue")`. Create a new issue for this API to become stable and add it as part of the annotation here.